packages/react-native-tracer-provider/native-src/android/app/dependencies.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+  // TODO keep this in sync with the version from the core/ package
+  api "io.embrace:embrace-android-sdk:6.9.1"
```

Comment:
would it make sense to have some tooling around bumping the android SDK version to keep them all in sync?  if we don't have tooling i would do the following:

* add a comment like this one in `packages/core/android/gradle.properties`
* mention all places that need to be changed in `DEVELOPING.md`

Reply:
I got blocked on what I was working on so tackled this instead, here's what I was going to go with: https://github.com/embrace-io/embrace-react-native-sdk/pull/55